### ISSUE TITLE
BattlefieldGuard module broken. Not kicking players when falling out of the arena.

### ISSUE THREAD

**Issue by mibby** [slipcor/pvparena#226]
@slipcor Kind of an important bug - breaks run arena. Players aren't kicked out when falling out of the world.

Debug
http://paste.ubuntu.com/18680741/

Paper dev 810
PVPArena dev 159
BattlefieldGuard 1.3.2.51

Arena config.
http://paste.ubuntu.com/18680837/

**Comment by slipcor** [comment]
Did you maybe have some protection in place about players not receiving damage? That is what battlefieldguard tries to do :) More specifically the arena region, it tries to apply 1000 damage. Because of the config "leavedeath" ;)

**Comment by mibby** [comment]
Not that I know of. Falling through the void outside of arenas damages me fine. It use to work fine, didn't make any changes to the arena config that I can recall.

Config
http://paste.ubuntu.com/20018282/

**Comment by slipcor** [comment]
no clue maybe they applied the logic "<1" too? No idea :/

**Comment by slipcor** [comment]
so the only way to fix this now is to set "leaveDeath" to false, so they get plainly kicked. Or them fixing their logic ;)

**Comment by slipcor** [comment]
No, it is just not "forcefully trying to kill a player (and failing)" but "kicking them out" :)
